internal/commands/test.go

```diff
@@ -144,19 +144,23 @@ func NewTestCommand(ctx context.Context) *cobra.Command {
 
 // GetResult returns the result of testing the structured data against their policies
 func GetResult(ctx context.Context, namespace string, input interface{}, compiler *ast.Compiler) (CheckResult, error) {
-	warnings, err := runRules(ctx, namespace, input, warnQ, compiler)
+	var successes []error
+	warnings, warnSuccesses, err := runRules(ctx, namespace, input, warnQ, compiler)
 	if err != nil {
 		return CheckResult{}, err
 	}
+	successes = append(successes, warnSuccesses...)
 
-	failures, err := runRules(ctx, namespace, input, denyQ, compiler)
+	failures, failureSuccesses, err := runRules(ctx, namespace, input, denyQ, compiler)
```

Comment:
This naming really confuses me, as Fail and Succeed are the opposites of each other. 
Is there a name which more clearly communicates what this variable contains?

Reply:
I fixed to use `totalSuccesses` for aggregation and `successes` to fail and deny rules.